common/workunit/workunit.cpp

```diff
@@ -13611,6 +13611,34 @@ extern WORKUNIT_API void addTimeStamp(IWorkUnit * wu, StatisticScopeType scopeTy
     wu->setStatistic(queryStatisticsComponentType(), queryStatisticsComponentName(), scopeType, scopestr, kind, NULL, getTimeStampNowValue(), 1, 0, StatsMergeAppend);
 }
 
+extern WORKUNIT_API void setGraphCost(IWorkUnit * wu, __int64 cost, StatisticScopeType scopeType, const char * scope, StatisticKind kind, unsigned wfid)
+{
+    StringBuffer scopestr;
+    if (wfid && scope && *scope)
```

Comment:
I'm not sure we really want this code to combine the graph id and the wfid.  It was added in other places to ease transition, to the new scheme, but it is better if the caller passes the complete scope.  E.g. in the one place this is currently called graphScope already contains the scope that is calculated here.

Reply:
It is probably easier to directly call wu->setStatistic than have an intermediate function.